### PR TITLE
PCM-3025: Added query to URL

### PR DESCRIPTION
**Pull request by jtrantin**
@vrathee @gandhikeyur Please review.

https://projects.engineering.redhat.com/browse/PCM-3025

I also added a fix for the same problem as in PCM-2943, and some more small fixes usually related to the advanced search.

### CODE REVIEW

File: app/assets/sass/elements/_base.scss [hunk 1]
```diff
@@ -152,4 +152,11 @@ a { color: $link-color; }
 		color: #afafaf;
 		margin-right: 8px;
 	}
-}
\ No newline at end of file
+}
+
+.spinner.ng-animate {
```
Comment (jtrantin) [inline app/assets/sass/elements/_base.scss]: Fixes 'hanging' spinners, as in PCM-2943.

**Comment by renujhamtani** [comment]
We would merge this after the PCM prod push.

**Comment by jtrantin** [comment]
OK. If I manage to finish PCM-3024 by then, we can do it in one PR, since I rewrote some of the code introduced here.

**Comment by jtrantin** [comment]
Since most of the code here is rewritten/removed in the next issue I am working on, I will submit the other one which will include both JIRAs (PCM-3025 + PCM-3024). Let me know if that is a problem.

**Pull request closed without merging**